pydicom/filereader.py

```diff
@@ -39,14 +42,17 @@
 from pydicom.fileutil import read_undefined_length_value
 from struct import Struct, unpack
 from sys import byteorder
+
 sys_is_little_endian = (byteorder == 'little')
 
 
 class DicomIter(object):
     """Iterator over DICOM data elements created from a file-like object
     """
+
     def __init__(self, fp, stop_when=None, force=False):
-        """Read the preamble and meta info and prepare iterator for remainder of file.
+        """Read the preamble and meta info and prepare iterator for remainder
```

Comment:
keep the method summary within one line. Then if necessary keep a longer explanation.

"""Parses metadata and sets the iterator

Parses the preamble and meta data and sets the ..
"""